pkgs/development/python-modules/torch/default.nix

```diff
@@ -111,7 +111,7 @@ let
   brokenConditions = attrsets.filterAttrs (_: cond: cond) {
     "CUDA and ROCm are not mutually exclusive" = cudaSupport && rocmSupport;
     "CUDA is not targeting Linux" = cudaSupport && !stdenv.isLinux;
-    "Unsupported CUDA version" = cudaSupport && (cudaPackages.cudaMajorVersion != "11");
+    "Unsupported CUDA version" = cudaSupport && !(cudaPackages.cudaMajorVersion == "11" || cudaPackages.cudaMajorVersion == "12");
```

Comment:
Maybe `!(builtins.elem cudaMajorVersion [ "11" "12" ])`

Reply:
Fixed in latest